js/blogtini.js

```diff
@@ -314,21 +341,24 @@ async function main() {
   if (!STORAGE.base)
     STORAGE.base = base
 
-  tmp = yml.load(await fetcher(`${state.top_dir}config.yml`))
-  if (tmp)
+  const configYamlUrl = `${SITE_ROOT_BASE_URL}config.yml`
+  tmp = yml.load(await fetcher(configYamlUrl))
+  if (tmp) {
     cfg = { ...cfg, ...tmp } // xxx deep merge `sidebar` value hashmap, too
+  }
 
 
   log({
     filter_post, base: STORAGE.base, STORAGE_KEY, cfg, state,
   })
 
-  const prefix = cfg.repo === 'blogtini' ? state.pathrel : 'https://blogtini.com/'
-  // eslint-disable-next-line no-use-before-define
-  add_css(`${prefix}css/blogtini.css`) // xxxx theme.css
+  add_css(`${SITE_ROOT_BASE_URL}css/blogtini.css`) // xxxx theme.css
 
+  cleanUpInitialPayloadMarkup(document)
 
   document.getElementsByTagName('body')[0].innerHTML = `
+  const blogtiniMain = createBlogtiniStuffWrapper(document, 'blogtini-main')
+  blogtiniMain.innerHTML = `<!-- RBx htmlString main BEGIN -->
```

Comment:
Instead of using `innerHTML`, we can use `document.createElement('template').innerHTML` and clone and then `appendChild`. Otherwise (especially when using customElements) it mounts components more than once because of re-creating.  It was a head scratcher why I would have the double of components in the console.log to realize that the re-creation effectively drops and re-created the same.